mkosi/sandbox.py

```diff
@@ -98,8 +99,19 @@ def oserror(filename: str = "") -> None:
     raise OSError(ctypes.get_errno(), os.strerror(ctypes.get_errno()), filename or None)
 
 
+UNSHARE_EPERM_MSG = """
+mkosi was forbidden to unshare the user namespace. This probably means your distro has
+restricted unprivileged user namespace creation to reduce exposure to kernel bugs.
+For information about Ubuntu and AppArmor implementations of this restriction,
+see https://ubuntu.com/blog/ubuntu-23-10-restricted-unprivileged-user-namespaces.
+For other systems, research the kernel.unprivileged_userns_clone and
+user.max_user_namespaces sysctls.
+"""
```

Comment:
That's a pretty dense block of text. Let's make this a bit lighter so it stands out visually.
```suggestion
UNSHARE_EPERM_MSG = """\
mkosi was unable to unshare the user namespace. 

This probably means your distribution has restricted the creation of unprivileged user namespaces.

Please consult the mkosi man page, e.g. via "mkosi documentation" for workarounds.
"""

```
The specifics for workarounds should go into the manual, where they can be looked up properly and where we can add more text and more examples, e.g. a snippet people can copy.

Reply:
Sounds good - I think "forbidden" is better than "unable" though since we are specifically talking about `EPERM` here? Given what I learned in [this thread](https://github.com/systemd/mkosi/pull/3266#discussion_r1874779387).

Reply:
Even though it's `EPERM` I think "unable" is a nicer way of putting it, since it's not mkosi being singled out, but this being a flat policy that mkosi wasn't yet exempted from. What I'm saying is that "forbidden" sounds meaner than being collateral damage of wider security policy.